### PR TITLE
[LIKE 90% MODULAR] Nurse

### PR DESCRIPTION
**Pull request by axietheaxolotl**
## About The Pull Request

Adds the Nurse job.

ok

## How This Contributes To The Skyrat Roleplay Experience

Nurse as a job, will reduce the amount of do-nothing MDs that just take a slot to put on some sutures. Also, le RP job.

The idea of this, is that it adds a job, that can very easily add to the RP value of Medical. But if they screw off, then nothing of real value is lost, and an MD slot isn't lost.

## Changelog

:cl: Nanotrasen Medical Appointment Department
add: We've assigned nurses to help the medical department do their job.
/:cl:

### CODE REVIEW

**Comment by GoldenAlpharex** [comment]
I think you suffered the same fate I did when opening your PR.

File: modular_skyrat/modules/nurse/code/nurse.dm [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/datum/job/nurse
+	title = "Nurse"
+	department_head = list("Chief Medical Officer")
```
Comment (Iamgoofball) [inline modular_skyrat/modules/nurse/code/nurse.dm:4]: you need spawn points for these mofos, and by that I mean put this on the job:
``job_spawn_title = "Medical Doctor"``

Comment (axietheaxolotl) [reply modular_skyrat/modules/nurse/code/nurse.dm:4]: A'ight.

**Comment by axietheaxolotl** [comment]
why byond

why must you make me suffer

**Comment by SpaceVampire** [comment]
This really seems like pointless job bloat. If you think MD slots are being wasted, it would be much easier to just increase the number of them instead of adding an unnecessary role.

**Comment by Iamgoofball** [comment]
More actual jobs are fine with the pop we're getting, and if it serves as a "intro to medical" role we can stick the newbies in then rad.

File: modular_skyrat/modules/nurse/code/nurse.dm [hunk 2]
```diff
@@ -0,0 +1,64 @@
+/datum/job/nurse
```
Comment (tf-4) [inline modular_skyrat/modules/nurse/code/nurse.dm:1]: Could this not just be a doctor subtype?

**Review by tf-4 — COMMENTED** [review]
Kind of meh imo, It's solving a problem that I'm not entirely convinced really exists, at least not in a substantial enough way for an entire job to be made to solve it.
3 also seems like a lot of slots. 2 would be better imo.

**Comment by Iamgoofball** [comment]
i don't believe the system supports job subtypes

**Comment by tf-4** [comment]
> i don't believe the system supports job subtypes

we live in a fucking society

**Comment by nikothedude** [comment]
> More actual jobs are fine with the pop we're getting, and if it serves as a "intro to medical" role we can stick the newbies in then rad.

To be honest, I think a lot of the pop we get comes from off-duty roles, like assistant, and unless people are only playing assistant because MD is full (it isn't), a nurse job won't help with that

**Comment by gonenoculer5** [comment]
Unnessecary job bloat. Its a no from me dog. 👎

**Comment by softcerv** [comment]
> Unnessecary job bloat. Its a no from me dog. 👎

we are high enough pop that job bloat isn't a huge issue though

**Comment by ChungusGamer666** [comment]
Job bloat is always bad. 
If you won't add any new meaningful content, increase the medical doctor cap.

**Comment by softcerv** [comment]
>Job bloat is always bad.
If you won't add any new meaningful content, increase the medical doctor cap.
Edit: No, i don't care about high pop, this is extremely variable and is not enough justification to add poor quality, low effort "content".

Having more slots would be nice, but I don't think it would really solve the issue that adding nurse would solve.

1. Since you'd have more people working on shift as medical doctor more of those people would think it would be okay to not do their job because "I'm sure the other doctors will handle it." I've witnessed this on a very regular basis with medical.
2. Even if more slots are added, you are still having someone taking up a doctor slot just so that they can do nurse (e)rp, it's better just to funnel that into a separate job so that you have more people that want to play the game playing medical doctor.
3. This could also serve as a good entry role to medical that isn't paramedic, it'd be good for new players to have a low pressure way to get comfortable with medical before the fully dive in as a MD.

If we want to like actually punish people for ERPing and neglecting their job duty as medical, then I'd say nurse has less of a place, but I doubt that is ever going to happen, so it's probably better just to get a job to funnel those kinds of people into

**Comment by ChungusGamer666** [comment]
> > Job bloat is always bad.
> > If you won't add any new meaningful content, increase the medical doctor cap.
> > Edit: No, i don't care about high pop, this is extremely variable and is not enough justification to add poor quality, low effort "content".
> 
> Having more slots would be nice, but I don't think it would really solve the issue that adding nurse would solve.
> 
> 1. Since you'd have more people working on shift as medical doctor more of those people would think it would be okay to not do their job because "I'm sure the other doctors will handle it." I've witnessed this on a very regular basis with medical.
> 2. Even if more slots are added, you are still having someone taking up a doctor slot just so that they can do nurse (e)rp, it's better just to funnel that into a separate job so that you have more people that want to play the game playing medical doctor.
> 3. This could also serve as a good entry role to medical that isn't paramedic, it'd be good for new players to have a low pressure way to get comfortable with medical before the fully dive in as a MD.
> 
> If we want to like actually punish people for ERPing and neglecting their job duty as medical, then I'd say nurse has less of a place, but I doubt that is ever going to happen, so it's probably better just to get a job to funnel those kinds of people into

I am talking from a game design standpoint (the only one that matters). 
Players joining the round that want to contribute with **literally nothing** should join as assistant and ask the HoP for extra access, plain and simple. The code should not strive to accomodate this kind of people.
Adding a literal ERP role for medical is quite literally greenlighting, and arguably encouraging, shitty player behavior.
I also really dislike the implication that this creates, that being "nurses are useless sex slaves" even though they do most of the actual work IRL.
Newbies should join as assistant and ask for medical access, too. I find that adding bloaty jobs for "newbies" is just a very poor excuse - These people learn quick, and will at most take 3 rounds to be ready for an actual job.
I am trying very hard to find anything good that can come out of this, but i simply am unable to.

**Comment by softcerv** [comment]
> I am talking from a game design standpoint (the only one that matters).
Players joining the round that want to contribute with literally nothing should join as assistant and ask the HoP for extra access, plain and simple. The code should not strive to accomodate this kind of people.
Adding a literal ERP role for medical is quite literally greenlighting, and arguably encouraging, shitty player behavior.
I also really dislike the implication that this creates, that being "nurses are useless sex slaves" even though they do most of the actual work IRL.
Newbies should join as assistant and ask for medical access, too. I find that adding bloaty jobs for "newbies" is just a very poor excuse - These people learn quick, and will at most take 3 rounds to be ready for an actual job.
I am trying very hard to find anything good that can come out of this, but i simply am unable to.

That's how it **should** work, but you see people even in head jobs going off to private rooms and doing nothing most of the shift. This could be solved by cracking down on those people more, but that almost never ends up happening. 
While I did use ERP as an example, I think it'd be more akin to psychologist where it'd mostly be an RP job, that people end up ERPing in. There is a niche for people that want to do medical RP that goes untapped because Medical Doctor is a very mechanical role.

**Comment by ChungusGamer666** [comment]
> That's how it **should** work, but you see people even in head jobs going off to private rooms and doing nothing most of the shift. This could be solved by cracking down on those people more, but that almost never ends up happening. While I did use ERP as an example, I think it'd be more akin to psychologist where it'd mostly be an RP job, that people end up ERPing in. There is a niche for people that want to do medical RP that goes untapped because Medical Doctor is a very mechanical role.

RP without mechanics is a _myth_.
If there is no game master running the show (ideally in SS13, this GM is the code), everything will just devolve to godmoding. Psychologist is part of the problem and i don't find it to be a good justification to keep adding to the problem. 
So, my stance is pretty simple: Adding jobs purely for the sake of roleplay hurts the game, detracting players from the actual mechanics (which may result in worse rounds for the entire crew).

**Pull request closed without merging**